test/rewards/AutoCompoundingRewards.ts

```diff
@@ -189,11 +189,22 @@ describe('AutoCompoundingRewards', () => {
 
     describe('management', () => {
         const MIN_LIQUIDITY_FOR_TRADING = toWei(1_000);
-        const TOTAL_DURATION = duration.days(10);
         const TOTAL_REWARDS = toWei(10_000);
         const INITIAL_USER_STAKE = toWei(10_000);
 
         const START_TIME = 1000;
+        const FLAT_TOTAL_DURATION = duration.days(10);
+        const EXP_DECAY_HALF_LIFE = duration.days(560);
+        const EXP_DECAY_MAX_DURATION = EXP2_INPUT_TOO_HIGH.mul(EXP_DECAY_HALF_LIFE).sub(1).ceil().toNumber();
+
+        const END_TIME = (distributionType: RewardsDistributionType) =>
```

Comment:
Since these are functions, they shouldn't be named as `CONSTANTS`

Reply:
done